package/MDAnalysis/analysis/align.py

```diff
@@ -630,8 +630,9 @@ def rms_fit_trj(
         tol_mass=0.1,
         strict=False,
         force=True,
-        quiet=False,
+        verbose=None,
```

Comment:
The docstring for `_set_verbose` describes in details what is the procedure to follow at the end of the deprecation phase.